galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/internal/DockerManagerImpl.java

```diff
@@ -55,7 +55,7 @@
  */
 @Component(service = { IManager.class })
 public class DockerManagerImpl extends AbstractManager implements IDockerManagerSpi {
-    protected final String               NAMESPACE = "docker";
+    protected final String               NAMESPACE = "Docker";
```

Comment:
Cant cahnge the namespace name as this is what the property names are formed from

Reply:
Good Spot Thankyou